### PR TITLE
Applied changes based on #7

### LINKED ISSUE #4: Update Android for AndroidX
The Android project should be updated to target Android 10

### LINKED ISSUE #7: List of changes to make project even better!
1- Add Xamarin Essentials
2- Set DebugType and DebugSymbols to true for Debug builds
3- Use Android 10 SDK & Android X
4- Use Aapt2 in android
5- Use d8 in android
6- Enable concurrent garbage collector in android
7- Remove Xamarin.Android.Support.* nuget packages
8- Enable Mono interpreter for debug mode of iOS project.

I'm open to submit a PR to any of these changes which are acceptable by the Prism team too.

### PR DESCRIPTION
**Pull request by ysmoradi**
Use Android 10 SDK & Android X
Use Aapt2 in android
Use d8 in android
Enable concurrent garbage collector in android
Remove Xamarin.Android.Support.* nuget packages
Enable Mono interpreter for debug mode of iOS project.

fixes #4 Update for Android X
fixes #7 Update standard project settings

### CODE REVIEW

**Comment by ysmoradi** [comment]
This PR also closes #4

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj [hunk 1]
```diff
@@ -49,8 +52,11 @@
     <WarningLevel>4</WarningLevel>
     <AndroidManagedSymbols>true</AndroidManagedSymbols>
     <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
+    <AndroidDexTool>d8</AndroidDexTool>
+    <AndroidEnableSGenConcurrent>true</AndroidEnableSGenConcurrent>
+    <AndroidUseAapt2>true</AndroidUseAapt2>
   </PropertyGroup>
-<!--/+:cnd:noEmit -->
+  <!--/+:cnd:noEmit -->
```
Comment (dansiegel) [inline Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:59]: revert any changes to the noEmit lines

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj [hunk 2]
```diff
@@ -14,23 +14,23 @@
...
     <JavaMaximumHeapSize />
     <JavaOptions />
-    <NuGetPackageImportStamp></NuGetPackageImportStamp>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
   </PropertyGroup>
-<!--/-:cnd:noEmit -->
-<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+  <!--/-:cnd:noEmit -->
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
```
Comment (dansiegel) [inline Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:33]: revert these changes

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-<!--/-:cnd:noEmit -->
+    <!--/-:cnd:noEmit -->
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-<!--/+:cnd:noEmit -->
+    <!--/+:cnd:noEmit -->
```
Comment (dansiegel) [inline Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:7]: revert these changes

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj (hunk 1 above)
Comment (amirabbaska) [reply Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:59]: B-)

Comment (amirabbaska) [reply Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:59]: (~) @):-

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj (hunk 2 above)
Comment (amirabbaska) [reply Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:33]: ;-)

File: Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj (hunk 3 above)
Comment (amirabbaska) [reply Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:7]:
```
:-h 
```

Comment (amirabbaska) [reply Xamarin.Forms/BlankApp/BlankApp.Android/BlankApp.Android.csproj:7]: - (#)

**Comment by ysmoradi** [comment]
If someone has applied requested changes, I'm ok with that. But within 1 or 2 hours I'm able to do whatever required to make things work.
I'm busy on another task at the moment )-:

**Comment by ysmoradi** [comment]
I applied the requested changes. Note that there is a space before
```xml
<!--/-:cnd:noEmit -->
```
in the iOS project which is not side effect of my changes, but I'm ready to delete those extra spaces if you think this is a good idea!

**Comment by ysmoradi** [comment]
> You need to properly rebase this

I performed a rebase. Don't know If I performed correctly or not.
I also submitted a clean PR #15

**Comment by dansiegel** [comment]
not sure what happened to this branch... this looks like you did a merge pull rather than a rebase pull. You can tell because you have the commits that were added after your initial PR and about 10x the number of new lines ;)

closing this to focus on your fixed PR
